Client/Frontend/Home/TopSitesPanel.swift

```diff
@@ -10,6 +10,7 @@ import Storage
 private let log = Logger.browserLogger
 
 private let ThumbnailIdentifier = "Thumbnail"
+private let FrecencyLimit = 20
```

Comment:
Point still stands: we should figure out the max number of tiles we could show on this screen, and just use that as the limit.